sources/packages/web/src/components/generic/BodyHeader.vue

```diff
@@ -1,24 +1,26 @@
 <template>
-  <v-row no-gutters>
-    <v-col cols="auto" class="mr-2">
-      <dynamic-header
-        :title="fullTitleMessage"
-        :level="titleHeaderLevel"
-        class="category-header-large color-blue"
-      />
-    </v-col>
-    <v-col cols="auto" class="mr-2">
-      <slot name="status-chip"></slot>
-    </v-col>
-    <v-col>
-      <slot name="actions"></slot>
-    </v-col>
-  </v-row>
-  <v-row no-gutters class="mb-2">
-    <v-col>
-      <slot name="subtitle">{{ subTitle }}</slot>
-    </v-col>
-  </v-row>
+  <div>
```

Comment:
Added a div at this level to prevent warnings where a style was applied to the component, which caused issues because the component template had two components.
This reduces the number of browser warnings.